mycli/main.py

```diff
@@ -846,7 +846,7 @@ def format_output(self, title, cur, headers, expanded=False,
             if hasattr(cur, 'description'):
```

Comment:
I think we should move the `FIELD_TYPES.get(col[1], text_type)` to the "sanitize" function.

Reply:
@meeuw Good thinking :) - done.